### PR TITLE
Add send mail to developer

### PR DESCRIPTION
**Pull request by php-schubser**

### CODE REVIEW

File: src/Classes/SendMail.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    $shopVersion = ShopInfo::getModifiedVersion();
+    $message .= "<br />Modified version: " . $shopVersion . '<br />';
+    $message .= 'Browser: ' . $_SERVER['HTTP_USER_AGENT'] . '<br />';
+    $message .= 'PHP version: ' . phpversion() . '<br />';
+    $headers[] = 'MIME-Version: 1.0';
+    $headers[] = 'Content-type: text/html; charset=iso-8859-1';
+    $headers[] = 'From: '.$from.' <'.$fromEmail.'>';
+    if (mail($to, $subject, $message, implode("\r\n", $headers))) {
+      Notification::pushFlashMessage(
+        array(
```
Comment (RobinTheHood) [inline src/Classes/SendMail.php:45]: Looks nice, but can you use short version of Array. Only [...]

File: src/Classes/SendMail.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          "text" => "Erfolg: Die Nachricht wurde erfolgreich gesendet, wir werden so schnell wie m&ouml;glich antworten.", 
+          "type" => "info"
```
Comment (RobinTheHood) [inline src/Classes/SendMail.php:47]: You can use ö because we use UTF8. 😊

File: src/Classes/IndexController.php [hunk 3]
```diff
@@ -89,6 +89,10 @@ public function invoke()
                 $this->invokeSelfUpdate();
                 break;
 
+            case 'reportProblem':
```
Comment (RobinTheHood) [inline src/Classes/IndexController.php:92]: Nice! It‘s cool that you add this to the Controller. 👍👍

File: src/Classes/SendMail.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+class SendMail
+{
+  public function sendFeedback() 
+  {
+    $fromEmail = ArrayHelper::getIfSet($_POST, 'email', '');
+    $from = ArrayHelper::getIfSet($_POST, 'name', '');
+    $message = ArrayHelper::getIfSet($_POST, 'message', '');
+    if ($fromEmail == '' || $from == '' || $message == '') {
+      Notification::pushFlashMessage(
+        array(
```
Comment (RobinTheHood) [inline src/Classes/SendMail.php:27]: Looks nice, but can you use short version of Array. Only [...]

File: src/Classes/SendMail.php [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+          "text" => "Warnung: Felder k&ouml;nnen nicht leer gelassen werden.",
```
Comment (RobinTheHood) [inline src/Classes/SendMail.php:28]: You can use ö because we use UTF8. 😊

File: src/Classes/SendMail.php [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    }
+    $to = 'info@module-loader.de';
+    $subject = 'Report problem';
+    $shopVersion = ShopInfo::getModifiedVersion();
+    $message .= "<br />Modified version: " . $shopVersion . '<br />';
+    $message .= 'Browser: ' . $_SERVER['HTTP_USER_AGENT'] . '<br />';
+    $message .= 'PHP version: ' . phpversion() . '<br />';
+    $headers[] = 'MIME-Version: 1.0';
+    $headers[] = 'Content-type: text/html; charset=iso-8859-1';
+    $headers[] = 'From: '.$from.' <'.$fromEmail.'>';
```
Comment (RobinTheHood) [inline src/Classes/SendMail.php:42]: Can you put a space before and after the . (Dot) operator 😊

File: src/Templates/ReportProblem.tmpl.php [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<?php if (!defined('LOADED_FROM_INDEX') || LOADED_FROM_INDEX != 'true') { die('Access denied.'); }?>
+
+<?php use RobinTheHood\ModifiedModuleLoaderClient\SendMail; ?>
+
+<?php 
+if (isset($_POST['send_mail'])) {
+    SendMail::sendFeedback();
+} ?>
```
Comment (RobinTheHood) [inline src/Templates/ReportProblem.tmpl.php:8]: Can you put logical code (line 3-8) in the your new Methode invokeReportProblem () in IndexController. I know that the IndexController has to much code. In the future we can split up the IndexController in Multiple Controllers. 😊

**Comment by RobinTheHood** [comment]
Maybe we can add the Domain to the E-Mail. 🤓🤔. ???

**Comment by php-schubser** [comment]
I'm finished with changes, I hope you like it  :smile:

**Pull request merged**